test/api-response.test.js

```diff
@@ -1,20 +0,0 @@
-/* eslint-env mocha */
```

Comment:
This should be g2g now, there was a systemic issue in PSI recently summarized here https://github.com/GoogleChrome/lighthouse/issues/9902#issuecomment-556414195

I think we can leave this test :smile:

Reply:
Hi @exterkamp many thanks for your review. 

Much better your proposal for the test definitions 😃.

I have added `api-response.test.js` again.

I have also reverted some changes that I mixed up 🤦‍♂️ (.gitignore and package.json). Sorry about that.

Hope now everything is working as expected.

Best regards.